internal/slicer/slicer.go

```diff
@@ -98,9 +111,12 @@ func Run(options *RunOptions) (*Report, error) {
 			if len(pathInfo.Arch) > 0 && !contains(pathInfo.Arch, arch) {
 				continue
 			}
+
 			if pathInfo.Kind != setup.GlobPath {
 				addKnownPath(targetPath)
+				installedPaths[targetPath] = true
```

Comment:
What about a path with `until: mutate` though? Should that path be in `installedPaths`?

Reply:
In my mind, it was installed so it is recorded in `installedPaths` but then it was removed so it will be recorded in `removedPaths`, then the filter will check both. What do you think? Or are you talking about concerns with the naming?